docs/customizations-api/javascript/capabilities.md

```diff
@@ -381,6 +381,24 @@ Target that allows customizing the [user profile page](https://docs.lumapps.com/
 
 - [Display a message above all pages of a certain type](./use-cases#display-a-message-above-all-pages-of-a-certain-type)
 
+### Organization chart
+
+Target that allows customizing the user organization chart.
+
+![Target user organization chart](./assets/target-organization-chart.png "Target user organization chart")
+
+#### Compatibility
+
+|               |                                                                                                                  |
+|---------------|------------------------------------------------------------------------------------------------------------------|
+| Target ID     | `user-profile-org-chart`                                                                                                        |
+| Description   | Target the organization chart section of a user's profile.                                                                                    |
+| Placements    | Compatible with placements: `ABOVE`, `UNDER`. Customizations will be added as siblings of the displayed user's organization chart. The displayed user's data can be retrieved using [context](./api#organization-chart-context) |
+| Disable       | No compatibility                                                                                                 |
+| Changing text | No compatibility                                                                                                 |
+
+**Note:** The `User space` feature flag must be enabled.
```

Comment:
We might want to add another note saying that this is compatible with `toRenderWithContext` and link this section to the other section that describes the context used.